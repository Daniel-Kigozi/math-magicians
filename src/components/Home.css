.home-container {
  display: flex;
  flex-direction: column;
  padding: 10% 20%;
}

.home-container h1 {
  font-size: 50px;
  text-align: left;
}

.home-container p {
  font-size: 30px;
  margin-top: 0;
}
