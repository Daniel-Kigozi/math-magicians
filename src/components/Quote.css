.quotes-container {
  display: flex;
  flex-direction: column;
  padding: 10% 20%;
}

.quotes-container h1 {
  font-size: 50px;
  text-align: left;
}

.quotes-container p {
  font-size: 30px;
  margin-top: 0;
}
