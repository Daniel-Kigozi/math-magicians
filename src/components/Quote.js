import React from 'react';

const Quote = () => (
  <div className="quotes-container">
    <h1>Quotes</h1>
    <p>
      Mathematics as an expression of the human mind reflects the active will,
      the contemplative reason, and the desire for aesthetic perfection.
      Its basic elements are logic and intuition, analysis and construction,
      generality and individuality.
      â€” Richard Courant, German-American mathematician -
    </p>

  </div>
);
export default Quote;
