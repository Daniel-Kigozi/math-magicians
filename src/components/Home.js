import React from 'react';
import './Home.css';

const Home = () => (
  <div className="home-container">
    <h1>Home page</h1>
    <p>
      Many who have had an opportunity of knowing any more about mathematics
      confuse it with arithmetic,
      and consider it an arid science. In reality, however,
      it is a science which requires a great amount of imagination.
    </p>
  </div>
);
export default Home;
