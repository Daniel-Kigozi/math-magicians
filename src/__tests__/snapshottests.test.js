import { render } from '@testing-library/react';
import Home from '../components/Home';
import Quote from '../components/Quote';

describe('Snapshot for Calculator and Home', () => {
  it('Snapshot should match component render', () => {
    const homepage = render(<Home />);
    expect(homepage).toMatchSnapshot();
  });
  it('Snapshot should match component render', () => {
    const quoterender = render(<Quote />);
    expect(quoterender).toMatchSnapshot();
  });
});
