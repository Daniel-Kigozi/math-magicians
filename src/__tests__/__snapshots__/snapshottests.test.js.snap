// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for Calculator and Home Snapshot should match component render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home-container"
      >
        <h1>
          Home page
        </h1>
        <p>
          Many who have had an opportunity of knowing any more about mathematics confuse it with arithmetic, and consider it an arid science. In reality, however, it is a science which requires a great amount of imagination.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home-container"
    >
      <h1>
        Home page
      </h1>
      <p>
        Many who have had an opportunity of knowing any more about mathematics confuse it with arithmetic, and consider it an arid science. In reality, however, it is a science which requires a great amount of imagination.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot for Calculator and Home Snapshot should match component render 2`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="quotes-container"
      >
        <h1>
          Quotes
        </h1>
        <p>
          Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. — Richard Courant, German-American mathematician -
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="quotes-container"
    >
      <h1>
        Quotes
      </h1>
      <p>
        Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. — Richard Courant, German-American mathematician -
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
